charts/rancher-webhook/1.0.0+up0.2.0/Chart.yaml

```diff
@@ -1,8 +1,10 @@
 annotations:
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/hidden: "true"
+  catalog.cattle.io/kube-version: '>= 1.16.0-0 < 1.22.0-0'
```

Comment:
Webhook has to run every time Rancher starts, so this should be not maxed out. It supports 1.24, but in the future has to support every version rancher supports. As far as specific versions go, it will get upgraded as soon as rancher is upgraded, so old versions don't matter.

Reply:
@cbron We are trying to enforce a closed range, so in this case I think what would make sense is to bump it to `< 1.24.0-0` as that covers what Rancher supports right now, and then update the constraint as new K8s versions are introduced into Rancher. Does that sound good?

Reply:
For this it doesn't matter at all, I guess it's sort of annoying but we can do it. Moving forward we would need to add one in the main repo.